great_expectations/expectations/expectation.py

```diff
@@ -2501,22 +2487,18 @@ class QueryExpectation(BatchExpectation, ABC):
             the expectation.
         runtime_keys (optional[tuple]): Optional. A tuple of the keys that can be used to control output but will
             not affect the actual success value of the expectation (such as result_format).
-        default_kwarg_values (optional[dict]): Optional. A dictionary that will be used to fill unspecified
-            kwargs from the Expectation Configuration.
         query (optional[str]): Optional. A SQL or Spark-SQL query to be executed. If not provided, a query must be passed
             into the QueryExpectation.
 
     --Documentation--
         - https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations
     """
 
-    default_kwarg_values: ClassVar[Dict] = {
-        "result_format": ResultFormat.BASIC,
-        "catch_exceptions": False,
-        "meta": None,
-        "row_condition": None,
-        "condition_parser": None,
-    }
+    result_format: Union[ResultFormat, ResultFormatDict] = ResultFormat.BASIC
+    catch_exceptions: bool = False
+    meta: Optional[dict] = None
+    row_condition: Optional[str] = None
+    condition_parser: Optional[str] = None
```

Comment:
These are defined on the parent `BatchExpectation` right? I don't think we need to duplicate